[2016-11-22 12:37:07] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ')'' in F:\xampp\htdocs\v-mast.mvc\www\app\Controllers\EventsController.php:1610
Stack trace:
#0 [internal function]: Exception\Handler->handleShutdown()
#1 {main} [] []
[2016-11-22 12:37:15] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ')'' in F:\xampp\htdocs\v-mast.mvc\www\app\Controllers\EventsController.php:1610
Stack trace:
#0 [internal function]: Exception\Handler->handleShutdown()
#1 {main} [] []
[2016-11-22 13:35:19] production.ERROR: exception 'ErrorException' with message 'Undefined index: chunksData' in F:\xampp\htdocs\v-mast.mvc\www\app\Controllers\EventsController.php:1464
Stack trace:
#0 F:\xampp\htdocs\v-mast.mvc\www\app\Controllers\EventsController.php(1464): Exception\Handler->handleError(8, 'Undefined index...', 'F:\\xampp\\htdocs...', 1464, Array)
#1 [internal function]: App\Controllers\EventsController->information('69')
#2 F:\xampp\htdocs\v-mast.mvc\www\system\Routing\Controller.php(240): call_user_func_array(Array, Array)
#3 F:\xampp\htdocs\v-mast.mvc\www\system\Routing\ControllerDispatcher.php(91): Routing\Controller->callAction('information', Array)
#4 F:\xampp\htdocs\v-mast.mvc\www\system\Routing\ControllerDispatcher.php(60): Routing\ControllerDispatcher->call(Object(App\Controllers\EventsController), Object(Routing\Route), 'information')
#5 F:\xampp\htdocs\v-mast.mvc\www\system\Routing\Router.php(957): Routing\ControllerDispatcher->dispatch(Object(Routing\Route), Object(Http\Request), 'App\\Controllers...', 'information')
#6 [internal function]: Routing\Router->Routing\{closure}('69')
#7 F:\xampp\htdocs\v-mast.mvc\www\system\Routing\Route.php(147): call_user_func_array(Object(Closure), Array)
#8 F:\xampp\htdocs\v-mast.mvc\www\system\Routing\Router.php(1025): Routing\Route->run()
#9 F:\xampp\htdocs\v-mast.mvc\www\system\Routing\Router.php(993): Routing\Router->dispatchToRoute(Object(Http\Request))
#10 F:\xampp\htdocs\v-mast.mvc\www\system\Foundation\Application.php(744): Routing\Router->dispatch(Object(Http\Request))
#11 F:\xampp\htdocs\v-mast.mvc\www\system\Foundation\Application.php(717): Foundation\Application->dispatch(Object(Http\Request))
#12 F:\xampp\htdocs\v-mast.mvc\www\shared\Http\ContentGuard.php(62): Foundation\Application->handle(Object(Http\Request), 1, true)
#13 F:\xampp\htdocs\v-mast.mvc\www\system\Session\Middleware.php(77): Shared\Http\ContentGuard->handle(Object(Http\Request), 1, true)
#14 F:\xampp\htdocs\v-mast.mvc\www\system\Cookie\Queue.php(49): Session\Middleware->handle(Object(Http\Request), 1, true)
#15 F:\xampp\htdocs\v-mast.mvc\www\system\Cookie\Guard.php(54): Cookie\Queue->handle(Object(Http\Request), 1, true)
#16 F:\xampp\htdocs\v-mast.mvc\www\vendor\stack\builder\src\Stack\StackedHttpKernel.php(23): Cookie\Guard->handle(Object(Http\Request), 1, true)
#17 F:\xampp\htdocs\v-mast.mvc\www\system\Foundation\Application.php(615): Stack\StackedHttpKernel->handle(Object(Http\Request))
#18 F:\xampp\htdocs\v-mast.mvc\www\app\Boot\Start.php(250): Foundation\Application->run()
#19 F:\xampp\htdocs\v-mast.mvc\www\public\index.php(37): require('F:\\xampp\\htdocs...')
#20 {main} [] []
[2016-11-22 13:37:55] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in F:\xampp\htdocs\v-mast.mvc\www\app\Controllers\EventsController.php:1425
Stack trace:
#0 [internal function]: Exception\Handler->handleShutdown()
#1 {main} [] []
